client/src/Components/Location/Location.js

```diff
@@ -2,18 +2,45 @@ import React, { Component } from 'react';
 import './Location.css';
 
 class Location extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {wifi: false, privacy: false, quiet: false, foodAndDrink: false};
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: !this.state[event.target.id]
```

Comment:
Never seen this syntax before! [Node supports it](http://node.green/#ES2015-syntax-object-literal-extensions). :+1: